Misc/NEWS.d/next/Library/2020-02-02-10-38-59.bpo-39350.8V69hi.rst

```diff
@@ -0,0 +1,3 @@
+Revert removal of ``fractions._gcd``. That removal caused breakage when
+using ``fractions.Fraction`` with integer-like values (for example NumPy
+integers).
```

Comment:
You may reuse my NEWS entry to avoid mentioning the private _gcd() method, I don't think that it matters here:

```
Fix regression in :class:`fractions.Fraction` if the numerator and/or the
denominator is an :class:`int` subclass.
```